### PR TITLE
add newsletter backoffice part

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

File: NewsletterAdminBundle/DisplayIcon/NewsletterStrategy.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+     * Perform the show action for a block
+     *
+     * @return string
+     */
+    public function show()
+    {
+        return $this->render('OpenOrchestraNewsletterAdminBundle:Block/Newsletter:showIcon.html.twig');
+    }
+
+
```
Comment (alavieille) [inline NewsletterAdminBundle/DisplayIcon/NewsletterStrategy.php]: remove this empty line

File: NewsletterAdminBundle/Resources/config/generate_form.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+parameters:
+    open_orchestra_newsletter.generate_form.newsletter.class: OpenOrchestra\NewsletterAdminBundle\GenerateForm\NewsletterStrategy
+
+services:
+    open_orchestra_newsletter.generate_form.newsletter:
+        class: %open_orchestra_newsletter.generate_form.newsletter.class%
+        tags:
+            - { name: open_orchestra_backoffice.generate_form.strategy }
```
Comment (itkg-alavieille) [inline NewsletterAdminBundle/Resources/config/generate_form.yml]: eof

File: NewsletterAdminBundle/Resources/config/icon.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+parameters:
+    open_orchestra_newsletter.icon.newsletter.class: OpenOrchestra\NewsletterAdminBundle\DisplayIcon\NewsletterStrategy
+
+services:
+    open_orchestra_newsletter.icon.newsletter:
+        class: %open_orchestra_newsletter.icon.newsletter.class%
+        tags:
+            - { name: open_orchestra_backoffice.display_icon.strategy }
```
Comment (itkg-alavieille) [inline NewsletterAdminBundle/Resources/config/icon.yml]: eof

**Comment by alavieille** [comment]
except for checkstyle  :+1:

**Pull request merged**